pkg/agent/api/delegatedidentity/v1/service_test.go

```diff
@@ -735,10 +795,19 @@ type FakeAttestor struct {
 	err       error
 }
 
+type FakeWorkloadPIDAttestor struct {
```

Comment:
nit: this should be moved before the FakeAttestor.Attest method to be alongside its own methods.